commands/triggers.go

```diff
@@ -42,6 +43,12 @@ when events from that source type occur.  Currently, only the ` + "`" + `schedul
 		Writer, aliasOpt("ls"), displayerType(&displayers.Triggers{}))
 	AddStringFlag(list, "function", "f", "", "list only triggers for the chosen function")
 
+	enable := CmdBuilder(cmd, RunTriggerToggle(true), "enable <triggerName>", "Enable a trigger", "Use `doctl serverless triggers enable <triggerName>` to enable a trigger", Writer)
+	AddStringFlag(enable, "output", "o", "", "Desired output format [text|json] (default \"text\")")
+
+	disable := CmdBuilder(cmd, RunTriggerToggle(false), "disable <triggerName>", "Disable a trigger", "Use `doctl serverless triggers disable <triggerName>` to disable a trigger", Writer)
+	AddStringFlag(disable, "output", "o", "", "Desired output format [text|json] (default \"text\")")
```

Comment:
doctl has a global `output` flag but I able to get it to work with this command, so I added another flag specific to this command. Any guidance here would be great.